tests/9-soc/murax/CMakeLists.txt

```diff
@@ -69,7 +69,6 @@ foreach(FREQ ${BASYS_FREQS})
         basys3-full_toplevel_${FREQ}.v
         Murax.v
       INPUT_IO_FILE basys3-full.pcf
-      SDC_FILE basys3-full_${FREQ}.sdc
```

Comment:
Why not merge the PCF with the SDC in the XDC here?

Reply:
I think this could be done as a followup cleanup PR, where we finally get all the constraints file unified.
This PR is intended to just make use of the XDC instead of SDC regarding clock constraints.